src/Admin/Admin.js

```diff
@@ -1,12 +1,21 @@
 import React, { useState } from "react"
 // import { useHistory } from "react-router-dom"
-import { useNavigate } from "react-router-dom";
+import { useNavigate } from "react-router-dom"
 import "../Admin/Admin.css"
 
 const Admin = ({ setAdminSelections }) => {
   const [selectedRestaurant, setSelectedRestaurant] = useState("")
   const [selectedAction, setSelectedAction] = useState("")
   const navigate = useNavigate()
+  const restaurantId = () => {
```

Comment:
We will have to make sure that this logic still works when we pull in the API that the backend built. I remember there being discussion about the way that they are assigning unique ID's